Update with the master haddocking website

zuzanaj opened the pull request.

amjjbonvin commented:
Zuzana, there are many conflicts in your pull request. Please pull first from the master before updating and pushing

amjjbonvin committed:
Added Ganana project

amjjbonvin committed:
Corrected link

amjjbonvin committed:
Replaced twitter by bluesky, added eScience center

amjjbonvin committed:
correction

amjjbonvin committed:
Added HADDOCK3 publication + logo to software page

amjjbonvin committed:
Updated haddock3 version

amjjbonvin committed:
Formatting

mgiulini committed:
first draft of HADDOCK24 protein glycan tutorial

mgiulini committed:
added result page and zip file

mgiulini committed:
corrected typo in HADDOCK3 prot glyc tutorial

mgiulini committed:
updated zip

mgiulini committed:
added bonus section

amjjbonvin committed:
Updated discoverer instructions

amjjbonvin committed:
Small edits

mgiulini committed:
addressed some comments

mgiulini committed:
changed bonus picture

mgiulini committed:
added rmsd values

mgiulini committed:
typos/grammar

mgiulini committed:
added link in haddock2.4 menu

mgiulini committed:
clarified bonus restraints

amjjbonvin committed:
Added preprint

amjjbonvin committed:
Added preprint

AnnaKravchenko committed:
add run time 

for full run and short run

mgiulini committed:
small clarifications

AnnaKravchenko committed:
1. remove old folder name 2. add info about raw_input.tolm

amjjbonvin committed:
Update index.md

amjjbonvin committed:
Merge pull request #705 from haddocking/prot_prot_adjustments

small clarifications in basic protein protein tutorial

amjjbonvin committed:
Merge pull request #704 from haddocking/sofia-workshop-mini-fix

Minor corrections antibody-antigen tuto for Sofia workshop

mgiulini committed:
Merge pull request #703 from haddocking/haddock24_protein_glycan

HADDOCK24 protein glycan tutorial

amjjbonvin committed:
Corrected typo

amjjbonvin committed:
Added publication

AnnaKravchenko committed:
Bonus on interface energetics with prot-on, haddock3-score and alascan (#706)

* bonus on interface energetics with prot-on, haddock3-score and alascan

* Update index.md

* add final image

* upd link to zip file

* final round of correcting typos

---------

Co-authored-by: Alexandre Bonvin <alexandre.bonvin@gmail.com>

amjjbonvin committed:
Update for BioExcel summerschool

amjjbonvin committed:
Updated publications

AnnaKravchenko committed:
Add link to developer’s guide to the antibody-antigen tutorial  (#709)

* Add link to developer’s guide as an alternative to haddock3-cfg -m ${module}

* Copy pre-computed folder description to SummerSchool section

---------

Co-authored-by: Alexandre Bonvin <alexandre.bonvin@gmail.com>

AnnaKravchenko committed:
Rename Prot-on resu to index.md

because it was renamed on accident in last commit

amjjbonvin committed:
Merge pull request #710 from haddocking/fix-tuto

Rename Prot-on resu to index.md

amjjbonvin committed:
Updated publications

amjjbonvin committed:
Updated publication

amjjbonvin committed:
Added news item about Merus

mgiulini committed:
added links to manual and to repo example

mgiulini committed:
Merge pull request #696 from haddocking/nb-ag-tutorial

Nb antigen tutorial

rvhonorato committed:
simplify setup instructions

rvhonorato committed:
update gemfile

rvhonorato committed:
add links to `haddock-restraints`

rvhonorato committed:
Update `haddock3/index.md`

rvhonorato committed:
Update `HADDOCK3/index.md`

rvhonorato committed:
Update `education/index.md`

rvhonorato committed:
Update `HADDOCK24/index.md`

rvhonorato committed:
Merge pull request #712 from haddocking/haddock-restraints

add links to `haddock-restraints`

rvhonorato committed:
review

rvhonorato committed:
Update README.md

Co-authored-by: Arha <kravchenko.anna.d@gmail.com>

rvhonorato committed:
Update README.md

Co-authored-by: Arha <kravchenko.anna.d@gmail.com>

rvhonorato committed:
Merge pull request #711 from haddocking/readme

simplify setup instructions

rvhonorato committed:
Update restraints.md

rvhonorato committed:
Update haddock-start.html

rvhonorato committed:
Update airs.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md-Bratislava

rvhonorato committed:
Update index.md-BioExcel2024

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
add `build.yml`

rvhonorato committed:
Merge pull request #715 from haddocking/ci

add action to check if the site is buildable

rvhonorato committed:
Apply suggestion from @VGPReys

Co-authored-by: Victor Reys <132575181+VGPReys@users.noreply.github.com>

rvhonorato committed:
Apply suggestion from @VGPReys

Co-authored-by: Victor Reys <132575181+VGPReys@users.noreply.github.com>

rvhonorato committed:
Apply suggestion from @VGPReys

Co-authored-by: Victor Reys <132575181+VGPReys@users.noreply.github.com>

rvhonorato committed:
Apply suggestion from @VGPReys

Co-authored-by: Victor Reys <132575181+VGPReys@users.noreply.github.com>

rvhonorato committed:
Apply suggestion from @VGPReys

Co-authored-by: Victor Reys <132575181+VGPReys@users.noreply.github.com>

rvhonorato committed:
Apply suggestion from @VGPReys

Co-authored-by: Victor Reys <132575181+VGPReys@users.noreply.github.com>

rvhonorato committed:
Apply suggestion from @VGPReys

Co-authored-by: Victor Reys <132575181+VGPReys@users.noreply.github.com>

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Update index.md

rvhonorato committed:
Merge pull request #714 from rvhonorato/gentbl

Update `GenTBL` links

rvhonorato committed:
add `publications.yml`

rvhonorato committed:
add `paper_citation.html`

this defines the formatting of the citation

rvhonorato committed:
add reusable references

rvhonorato committed:
update publication data

rvhonorato committed:
add social icons

rvhonorato committed:
add social formatting on `people.html`

rvhonorato committed:
update linting and data on `members.yml`

rvhonorato committed:
typo

rvhonorato committed:
Merge pull request #718 from haddocking/socials

add socials to `/people`

amjjbonvin committed:
Updated publications

rvhonorato committed:
Merge branch 'master' into 716-update-of-the-haddock24-server-reference-which-is-no-longer-advanced-online-but-nature-prot-19-32193241-2024

rvhonorato committed:
tweaks

rvhonorato committed:
remove `**`

rvhonorato committed:
Merge pull request #717 from haddocking/716-update-of-the-haddock24-server-reference-which-is-no-longer-advanced-online-but-nature-prot-19-32193241-2024

add logic to make citations re-usable

amjjbonvin committed:
Updated people page
- added new group picture
- added Danai to the group members

VGPReys committed:
adding links to socials

VGPReys committed:
re-order haddock3 tutorials

rvhonorato committed:
Merge pull request #719 from haddocking/vgpr-links

Adding links to socials for VGPReys

VGPReys committed:
Merge branch 'master' into rearrange-tutorials

VGPReys committed:
apply suggestions from review

VGPReys committed:
Merge pull request #720 from haddocking/rearrange-tutorials

Reordering of haddock3 tutorials

AnnaKravchenko committed:
Update members.yml

VGPReys committed:
note that they are the same

VGPReys committed:
Update education/HADDOCK3/index.md

VGPReys committed:
Merge branch 'master' into rearrange-tutorials

amjjbonvin committed:
Corrected author order

rvhonorato committed:
Merge pull request #721 from haddocking/socials_ak

add socials Anna K

VGPReys committed:
Update index.md

Apply suggestions

VGPReys committed:
Merge branch 'master' into rearrange-tutorials

VGPReys committed:
Merge pull request #722 from haddocking/rearrange-tutorials

Small upgrade on text to describe that basic tutorial is the antibody-antigen tutorial

AnnaEngel98 committed:
Update members.yml

rvhonorato committed:
Merge pull request #723 from haddocking/socials_AE

Update members.yml

amjjbonvin committed:
Updated haddock3 publication

amjjbonvin committed:
Added news item about EGI interview

amjjbonvin committed:
Update

amjjbonvin committed:
Should not be part of the repo - generated locally

amjjbonvin committed:
Restoring Gemfile.lock

AnnaKravchenko committed:
add link to anitbody-antigen Sofia video (#725)

* add link to anitbody-antigen Sofia video

* embed video

AnnaKravchenko committed:
Add link to AIRviewer3 to advanced protein-DNA tuto  (#727)

* add link to AIRviewer on python3, and related info

AnnaKravchenko committed:
upd zip links (#729)

from links to AnnaK's surfdrive to links of Alex’s surfdrive

amjjbonvin committed:
Updated publications

amjjbonvin committed:
Added publication (CAPRI editorial preprint)

amjjbonvin committed:
Updated HADDOCK3 installation instructions in tutorials

amjjbonvin committed:
Correct link to registration form

amjjbonvin committed:
Update index.md

amjjbonvin committed:
Updated install instructions

amjjbonvin committed:
Merge pull request #732 from haddocking/install-haddock3

Updated HADDOCK3 installation instructions in tutorials

AnnaKravchenko committed:
local set-up antibody-antigen tuto: new format, new instructions (#735)

amjjbonvin committed:
Updated publications

amjjbonvin committed:
Added new group member

iremkeser04 committed:
Protein-peptide Tutorial  (#731)

* protein-peptide tutorial - simplified version of the docking part of the SBM practical (its docking part)

---------

Co-authored-by: İrem Keser <iremkeser@Irem-MacBook-Pro.local>
Co-authored-by: Arha <kravchenko.anna.d@gmail.com>

AnnaKravchenko committed:
hide peptide tuto till polished (#736)

AnnaKravchenko committed:
polished till #Best-Practice section (#738)

dependabot[bot] committed:
Bump rexml from 3.4.1 to 3.4.2

Bumps [rexml](https://github.com/ruby/rexml) from 3.4.1 to 3.4.2.
- [Release notes](https://github.com/ruby/rexml/releases)
- [Changelog](https://github.com/ruby/rexml/blob/master/NEWS.md)
- [Commits](https://github.com/ruby/rexml/compare/v3.4.1...v3.4.2)

---
updated-dependencies:
- dependency-name: rexml
  dependency-version: 3.4.2
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

ntxxt committed:
Add links for Xiaotong Xu

rvhonorato committed:
Merge pull request #740 from ntxxt/master

Add links for Xiaotong Xu

AnnaKravchenko committed:
Polished final version of protein-peptide tuto (#741)

* polished text, second half"

AnnaKravchenko committed:
added missing img (#743)

amjjbonvin committed:
Added two news items

amjjbonvin committed:
Updated link

amjjbonvin committed:
Style

amjjbonvin committed:
Style

amjjbonvin committed:
Some formatting

amjjbonvin committed:
Removed unused instruction and clarified a point

AnnaKravchenko committed:
Protein-peptide basic [ready to be published] (#744)

* typos, small fixes

* Update index.md with Anna’s comments

* add link to zip on surfdrive of Alex

AnnaKravchenko committed:
Update index.md (#749)

amjjbonvin committed:
updated publication and group members

amjjbonvin committed:
Added picture

amjjbonvin committed:
Added new master student

AnnaKravchenko committed:
add MDM2 missing residues (#750)

rerun with correct protein, update output files

AnnaKravchenko committed:
Add AIRviewer3.py directly to haddock.io  (#733)

and remove older AIRviewer version from the archive

amjjbonvin committed:
Updated surfdrive download files

amjjbonvin committed:
Merge pull request #752 from haddocking/surfdrive

Updated surfdrive download files

amjjbonvin committed:
Added permanent links for Prot-ON results

rvhonorato committed:
Merge pull request #739 from haddocking/dependabot/bundler/rexml-3.4.2

Bump rexml from 3.4.1 to 3.4.2

amjjbonvin committed:
Updated group members

AnnaKravchenko committed:
add missing s to peptides (#754)

AnnaKravchenko committed:
Regroup HADDOCK2.4 tutorials (#755)

* regroup HADDOCK2.4 tutorials

amjjbonvin committed:
Added preprint

amjjbonvin committed:
Added news item about Anna's poster prize

amjjbonvin committed:
Updated publication

dkotzamp committed:
Update simulation tutorial

amjjbonvin committed:
Merge pull request #756 from dkotzamp/update-simulation-tutorial

Update simulation tutorial to use CHARMM36m and add terminal capping instructions

dkotzamp committed:
Fix typos and update content in tutorials
- Updated cryo-EM description to reflect near-atomic resolution
- Removed AlphaFold2 reference from modelling.md since AF model now exists
- Some typo corrections
Note: The CPORT server appears to be unavailable - Should we update the tutorial to replace it with WHISCY?

dkotzamp committed:
Merge branch 'master' into update-tutorials

amjjbonvin committed:
Updated group members

amjjbonvin committed:
Updated course link to nmrbox

dkotzamp committed:
Merge pull request #757 from dkotzamp/update-tutorials

Tutorial corrections

dkotzamp committed:
charmm36 ff requirement for gmx pdb2gmx

dkotzamp committed:
Update simulation.md

dkotzamp committed:
MOLMOD variables

dkotzamp committed:
Merge pull request #758 from dkotzamp/update-tutorials

charmm36 ff requirement for gmx pdb2gmx

AnnaKravchenko committed:
SBM 2026 updates  (#759)

* upd index, md and docking pages

AnnaKravchenko committed:
SBM minor 2026 (#760)


* info-prompt to check CPU load

AnnaKravchenko committed:
AF3 bonus for sbm course  (#761)

* AF3 bonus, links to resources

amjjbonvin committed:
Updated group members

amjjbonvin committed:
Merge pull request #762 from haddocking/group-update

Updated group members

amjjbonvin committed:
Updated instructions for ASM HCP/AI school in Japan

amjjbonvin committed:
updated group members

VGPReys committed:
adriatic edition updates

VGPReys committed:
adriatic edition updates

VGPReys committed:
fix env name

VGPReys committed:
Merge pull request #763 from haddocking/bioexcel-adriatic-ljubljana

BioExcel Adriatic Edition modification to Antibody-Antigen tutorial

amjjbonvin committed:
Added news item about new EU project

amjjbonvin committed:
Added news item about NWO compute grant

AnnaEngel98 committed:
powerfit webserver update

AnnaEngel98 committed:
CLI update

AnnaEngel98 committed:
update to integrative tutorial

AnnaEngel98 committed:
add date

rvhonorato committed:
redesign `people.html`

rvhonorato committed:
add `url` entry

rvhonorato committed:
Merge pull request #766 from haddocking/add-personal-link

add `url` entry

rvhonorato committed:
Merge branch 'master' into people-layout-tweaks

rvhonorato committed:
Merge pull request #765 from haddocking/people-layout-tweaks

redesign `people.html`

Comp-era committed:
Add social links for Shantanu Khatri

rvhonorato committed:
add x icon

rvhonorato committed:
update `people.html` to render X

rvhonorato committed:
add example to `members.yml`

rvhonorato committed:
Merge pull request #768 from haddocking/add-x

add X to socials

hankurt committed:
Add LinkedIn profile for Han Kurt

rvhonorato committed:
Merge branch 'master' into Comp-era-patch-1

rvhonorato committed:
Merge pull request #767 from haddocking/Comp-era-patch-1

Add social links for Shantanu Khatri

rvhonorato committed:
Merge pull request #769 from haddocking/hankurt-patch-1

Add LinkedIn profile for Han Kurt

rvhonorato committed:
add logic to transition member into pet

rvhonorato committed:
add extra fields

rvhonorato committed:
add placeholder pet picture

rvhonorato committed:
typo

AnnaEngel98 committed:
Merge branch 'master' into powerfit_updates

AnnaEngel98 committed:
comments

AnnaEngel98 committed:
typo

dkotzamp committed:
Merge pull request #764 from haddocking/powerfit_updates

powerfit webserver update

AnnaEngel98 committed:
update link

rversin committed:
Merge pull request #774 from haddocking/update_link

update link

AnnaEngel98 committed:
update deadline date

rversin committed:
Merge pull request #775 from haddocking/update_date

update deadline date

AnnaEngel98 committed:
small edits

AnnaEngel98 committed:
Merge pull request #776 from haddocking/update_rna_poll

Small edits to RNA-Poll-III integrative tutorial

AnnaEngel98 committed:
updates from pymol

AnnaEngel98 committed:
Update education/NMRMolmod/index.md

Co-authored-by: Raphaelle Versini <rapha.versini@gmail.com>

rversin committed:
Merge pull request #778 from haddocking/nmrmolmod_pymol

updates from pymol

rversin committed:
update question 19

AnnaEngel98 committed:
Merge pull request #779 from haddocking/change_Q19

update question 19

amjjbonvin committed:
Updated videos on education page, added publication

amjjbonvin committed:
Updated youtube links

amjjbonvin committed:
Added news item about BioExcel conference

amjjbonvin committed:
Updated haddock2.5 version and list of changes

amjjbonvin committed:
Added visiting PhD student to the people page

amjjbonvin committed:
Update changes.md

amjjbonvin committed:
Added new visiting PhD student to group

rvhonorato committed:
Merge branch 'master' into pets

rvhonorato committed:
Merge pull request #771 from haddocking/pets

add pets easter egg

amjjbonvin committed:
Updated publications

AnnaKravchenko committed:
Update basic prot-prot server tuto (#773)

* upd text and images; add more info about glucose transport; add reviews’ comments

amjjbonvin committed:
Updated group picture

amjjbonvin committed:
Added publication

amjjbonvin committed:
Merge pull request #780 from haddocking/haddock2.5

Updated haddock2.5 version and list of changes

amjjbonvin committed:
Updated HADDOCK3 antibody-antigen tutorial for BioExcel summerschool

amjjbonvin committed:
Update

amjjbonvin committed:
Updated publications

amjjbonvin committed:
Updated HADDOCK3 antibody-antigen tutorial with data from HADDOCK3 2026.5.0

amjjbonvin committed:
Updated group members
